ResearchKit/Common/ORKAnswerFormat.m

```diff
@@ -2210,4 +2302,8 @@ -(BOOL)isEqual:(id)object {
             _useCurrentLocation == castObject.useCurrentLocation);
 }
 
+- (NSString *)stringForAnswer:(id)answer {
+    return ABCreateStringWithAddressDictionary(((ORKPlacemark *)answer).addressDictionary, NO);
```

Comment:
Just a reminder, we are shifting away from `ORKPlacemark` by creating a custom `ORKLocation` object:

https://github.com/YuanZhu-apple/ResearchKit/blob/master_location_question_enhancement/ResearchKit/Common/ORKResult.h#L1259-L1283